dashboard/src/app/ide/ide.controller.js

```diff
@@ -81,6 +81,14 @@ class IdeCtrl {
 
       let promise = cheWorkspace.fetchWorkspaces();
 
+      if ($routeParams.showLogs) {
+        // remove action from path
+        $location.url('/ide/' + this.selectedWorkspaceName, false);
```

Comment:
If I understand corectly, we should add geter which returns the URL depends on workspace name (it can be near this one https://github.com/eclipse/che/blob/master/dashboard/src/components/api/che-workspace.factory.js#L294) to factory and use it. Because we have a several places are needed  relative IDE url. 

Then, for contextPath(https://github.com/eclipse/che/blob/master/dashboard/src/app/ide/ide.service.js#L258
), we should add a property like https://github.com/eclipse/che/blob/master/dashboard/src/assets/branding/product.json#L6

@benoitf Am I right?